users/views.py

```diff
@@ -56,10 +59,39 @@ def UserCreateConfirmView(request, token):
         return redirect('users:signup')
```

Comment:
You're right. There are a few more things that need to be updated, this is just one. But I will fix this one.